libfwupdplugin/fu-path.c

```diff
@@ -519,6 +519,12 @@ fu_path_from_kind(FuPathKind path_kind)
 			return g_steal_pointer(&localtime);
 		return g_strdup("/etc/localtime");
 	}
+	/* /sys/kernel/debug */
```

Comment:
I'm really hesitant about this one; we can use debugfs if lockdown and SecureBoot is enabled?

Reply:
It is only for the test suite, right?

Reply:
> It is only for the test suite, right?

Yes